auacm/app/modules/submission_manager/views.py

```diff
@@ -69,3 +69,44 @@ def submit():
     return serve_response({
         'submissionId': attempt.job
     })
+
+
+@app.route('/api/submit/job/<int:job_id>')
+def get_submit(job_id):
+    """Get the current status of a submission based on its id"""
+    submit = (session.query(models.Submission)
+              .filter(models.Submission.job == job_id).first())
+
+    if not submit:
+        return serve_error('Submission with job id ' + str(job_id) +
+                           ' not found', 401)
+
+    return serve_response(submit.to_dict())
+
+
+@app.route('/api/submit/user/<username>')
+@app.route('/api/submit/user/<username>/<int:limit>')
```

Comment:
Alternatively, we could do something like `/api/me/submissions` and possibly even `/api/users/<username>/submissions`.